src/main/java/com/merge/api/core/MergeApiApiError.java

```diff
@@ -0,0 +1,49 @@
+/**
+ * This file was auto-generated by Fern from our API Definition.
+ * 
+ * NOTE: Do NOT use this class. `ApiError` is the class that should
+ * be used. This class is just included for backwards compatibility
+ * purposes.
+ */
+package com.merge.api.core;
+
+/**
+ * This exception type will be thrown for any non-2XX API responses.
+ */
```

Comment:
you could consider adding `@Deprecated` here, which might catch someone's eye a bit easier